docs/fastdds/dds_layer/core/waitsets/waitsets.rst

```diff
@@ -36,9 +37,9 @@ application main loop.
    :end-before: //!
    :dedent: 4
 
-Calling the wait operation on the wait-set will block the calling thread if the trigger value of
-all the conditions attached to it are false.
-The thread will wake up, and the wait operation will return OK, whenever the trigger value of any
+Calling the |WaitSet::wait-api| operation on the :ref:`api_pim_waitset` will block the calling thread
+if the trigger value of all the conditions attached to it are false.
+The thread will wake up, and the |WaitSet::wait-api| operation will return OK, whenever the trigger value of any
```

Comment:
I suggest using `RETCODE_OK` even though the return codes are not still in the API reference. It is also what is shown in the example. This will probably allow us to remove the ok word from the spelling (and maybe RETCODE_OK would be needed).